docs/source/integrations/burrow.rst

```diff
@@ -64,15 +64,85 @@ Schematically the interaction between different parts of the system looks as fol
 	From the Burrow EVM perspective such Service Contract is hosted in a Native external VM and is callable via the same interfaces as if it was deployed at some special address in the EVM itself.
 	These methods are used specifically for Iroha integration, so the address of the Service Contract can only be found while working with it via Iroha.
 
-Current release of the Iroha EVM wrapper contains a single service contract deployed at the address `A6ABC17819738299B3B2C1CE46D55C74F04E290C` (the last 20 bytes of the *keccak256* hash of the string *ServiceContract*) which exposes 2 methods to query Iroha assets balances and transfer assets between accounts.
+Current release of the Iroha EVM wrapper contains a single service contract deployed at the address `A6ABC17819738299B3B2C1CE46D55C74F04E290C` (the last 20 bytes of the *keccak256* hash of the string *ServiceContract*) which exposes the following Iroha commands and queries:
+
+.. list-table:: Integrated Commands/Queries
+   :widths: 3 35 35
+   :header-rows: 1
+
+   * - S No.
+     - Signature of exposed API methods
+     - Arguments
+   * - 1
+     - addAsset(string,string)
```

Comment:
Optionally: IMO it would be good to provide links in the table: https://iroha.readthedocs.io/en/main/develop/api/commands.html#add-asset-quantity
of course not external but internal relative links